deploy/operator.yaml

```diff
@@ -35,7 +35,7 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: istio-workspace
-          image: quay.io/maistra/istio-workspace:v0.0.3
+          image: localhost:32000/istio-workspace-images/istio-workspace:latest
```

Comment:
you sure? :)

Reply:
That is autogenerated and already reverted